tests/test_packages/test_skills/test_echo.py

```diff
@@ -20,160 +20,59 @@
-                message_id=1,
-                target=0,
-                performative=DefaultMessage.Performative.BYTES,
-                content=b"hello",
-            )
-            expected_envelope = Envelope(
-                to=self.agent_name,
-                sender="sender",
-                protocol_id=DefaultMessage.protocol_id,
-                message=DefaultSerializer().encode(message),
-            )
-            encoded_envelope = "{},{},{},{},".format(
-                expected_envelope.to,
-                expected_envelope.sender,
-                expected_envelope.protocol_id,
-                expected_envelope.message.decode("utf-8"),
-            )
-            encoded_envelope = encoded_envelope.encode("utf-8")
-
-            with open(Path(self.t, self.agent_name, "input_file"), "ab+") as f:
-                f.write(encoded_envelope)
-                f.flush()
-
-            time.sleep(2.0)
-            with open(Path(self.t, self.agent_name, "output_file"), "rb+") as f:
-                lines = f.readlines()
-
-            assert len(lines) == 2
-            line = lines[0] + lines[1]
-            to, sender, protocol_id, message, end = line.strip().split(b",", maxsplit=4)
-            to = to.decode("utf-8")
-            sender = sender.decode("utf-8")
-            protocol_id = PublicId.from_str(protocol_id.decode("utf-8"))
-            assert end in [b"", b"\n"]
-
-            actual_envelope = Envelope(
-                to=to, sender=sender, protocol_id=protocol_id, message=message
-            )
-            assert expected_envelope.to == actual_envelope.sender
-            assert expected_envelope.sender == actual_envelope.to
-            assert expected_envelope.protocol_id == actual_envelope.protocol_id
-            assert expected_envelope.message == actual_envelope.message
-            time.sleep(2.0)
-        finally:
-            process.send_signal(signal.SIGINT)
-            process.wait(timeout=20)
-            if not process.returncode == 0:
-                poll = process.poll()
-                if poll is None:
-                    process.terminate()
-                    process.wait(2)
-
-            os.chdir(self.t)
-            result = self.runner.invoke(
-                cli, [*CLI_LOG_OPTION, "delete", self.agent_name], standalone_mode=False
-            )
-            assert result.exit_code == 0
-
-    @classmethod
-    def teardown_class(cls):
-        """Teardowm the test."""
-        os.chdir(cls.cwd)
-        try:
-            shutil.rmtree(cls.t)
-        except (OSError, IOError):
-            pass
+        assert expected_envelope.to == actual_envelope.sender
+        assert expected_envelope.sender == actual_envelope.to
+        assert expected_envelope.protocol_id == actual_envelope.protocol_id
+        assert expected_envelope.message == actual_envelope.message
+        time.sleep(2.0)
+
+        os.chdir(self.t)
+        self.delete_agents(agent_name)
```

Comment:
We should remove the delete agents command, it serves no real purpose (we test for this command working elsewhere)

Reply:
Agree. It just was there before, I don't know it's real purpose too.